### ISSUE TITLE
ModuleNotFoundError: No module named 'torch_integral.tsp_solver.solver'，but I can't find the problem, please help,Thank you!

### ISSUE THREAD

**Issue by EIDorado1** [TheStageAI/TorchIntegral#32]
<img width="1188" alt="截屏2023-12-04 17 26 58" src="https://github.com/TheStageAI/TorchIntegral/assets/62056405/f42b570c-dd71-4d69-a2ec-8334af0713e0">
But I checked all the packages and versions and found no problems, the environment for requerments is all followed as requested, and the git packages are downloaded. After re-executing ’setup install‘, it shows that the compilation is complete and I can see the version 0.0.0 of the package, but as soon as I import the package, it generates the error shown in the picture.

**Comment by EIDorado1** [comment]
![Uploading 截屏2023-12-04 17.33.08.png…]()

**Comment by EIDorado1** [comment]
ModuleNotFoundError: No module named 'torch_integral.tsp_solver.solver'
![Uploading 截屏2023-12-04 17.33.08.png…]()

**Comment by EIDorado1** [comment]
Yes, I've tried it according to the documentation you've given and the methods documented in previous questions from people. Now I'm trying to debug the miniset project to learn the principles of this article, but after checking the environment I can't find what the problem is.
![Uploading 截屏2023-12-06 13.36.12.png…]()

**Comment by EIDorado1** [comment]
![Uploading 截屏2023-12-06 13.36.12.png…]()

**Comment by EIDorado1** [comment]
This is a question of the project you want to start and its implementation.
<img width="1512" alt="截屏2023-12-06 13 36 12" src="https://github.com/TheStageAI/TorchIntegral/assets/62056405/a9b5aa2e-0329-4192-8b53-8282e6df0eb4">

**Comment by EIDorado1** [comment]
Yes, I've tried it according to the documentation you've given and the methods documented in previous questions from people. Now I'm trying to debug the miniset project to learn the principles of this article, but after checking the environment I can't find what the problem is.
<img width="1512" alt="截屏2023-12-06 13 36 12" src="https://github.com/TheStageAI/TorchIntegral/assets/62056405/719c6e17-e7bb-468f-b814-596721c1203c">

**Comment by EIDorado1** [comment]
<img width="988" alt="截屏2023-12-06 13 42 44" src="https://github.com/TheStageAI/TorchIntegral/assets/62056405/fd7dcaeb-9059-4b22-ba0d-fdfdcf35799c">
I have performed all the environments and steps required by the README.md file. I have also executed the setup.py install command, and all the environments required by the requirements.txt file have been installed.

**Comment by EIDorado1** [comment]
python setup.py install

**Comment by b1n0** [comment]
Use `pip install ./`. It should help.

**Comment by EIDorado1** [comment]
<img width="1477" alt="截屏2023-12-06 13 52 20" src="https://github.com/TheStageAI/TorchIntegral/assets/62056405/159e55ba-15d8-44bf-bb86-2da268ca1840">
I ran this command and then ran minist to test it.
<img width="1206" alt="截屏2023-12-06 13 53 40" src="https://github.com/TheStageAI/TorchIntegral/assets/62056405/e2e3d7e2-55c4-4483-aac4-d7fc7cc09d49">
The problem still seems to exist.

**Comment by EIDorado1** [comment]
I'll try to create a brand new python environment and put it under another path, thank you so much!

**Comment by ArnoldMSU** [comment]
@EIDorado1 Hello! Did you solve this problem? We plan to close this issue if so
